pages/_document.js

```diff
@@ -29,7 +29,15 @@ export default class MyDocument extends Document {
 
           <link href='/static/normalize.css' rel='stylesheet' />
           <link href='/static/base.css' rel='stylesheet' />
-
+          <script src="https://unpkg.com/react/umd/react.production.js" crossorigin />
+          <script src="https://unpkg.com/react-dom/umd/react-dom.production.js" crossorigin />
```

Comment:
next.js is already enabling react so we don't need to provide react and react dome here.

Reply:
Done!